x-pack/plugins/cases/public/components/all_cases/columns_popover.test.tsx

```diff
@@ -13,173 +13,182 @@ import type { AppMockRenderer } from '../../common/mock';
 import { createAppMockRenderer } from '../../common/mock';
 import { ColumnsPopover } from './columns_popover';
 
-describe('ColumnsPopover', () => {
-  let appMockRenderer: AppMockRenderer;
+for (let i = 0; i < 500; i++) {
+  describe('ColumnsPopover', () => {
+    let appMockRenderer: AppMockRenderer;
 
-  beforeEach(() => {
-    appMockRenderer = createAppMockRenderer();
-    jest.clearAllMocks();
-  });
+    beforeEach(() => {
+      appMockRenderer = createAppMockRenderer();
+      jest.clearAllMocks();
+    });
 
-  const selectedColumns = [
-    { field: 'title', name: 'Title', isChecked: true },
-    { field: 'category', name: 'Category', isChecked: false },
-    { field: 'tags', name: 'Tags', isChecked: false },
-  ];
+    const selectedColumns = [
+      { field: 'title', name: 'Title', isChecked: true },
+      { field: 'category', name: 'Category', isChecked: false },
+      { field: 'tags', name: 'Tags', isChecked: false },
+    ];
 
-  it('renders correctly a list of selected columns', async () => {
-    appMockRenderer.render(
-      <ColumnsPopover selectedColumns={selectedColumns} onSelectedColumnsChange={() => {}} />
-    );
+    it('renders correctly a list of selected columns', async () => {
+      appMockRenderer.render(
+        <ColumnsPopover selectedColumns={selectedColumns} onSelectedColumnsChange={() => {}} />
+      );
 
-    userEvent.click(await screen.findByTestId('column-selection-popover-button'));
+      userEvent.click(await screen.findByTestId('column-selection-popover-button'));
 
-    expect(await screen.findByTestId('column-selection-popover')).toBeInTheDocument();
+      expect(await screen.findByTestId('column-selection-popover')).toBeInTheDocument();
+      expect(
+        await screen.findByTestId(`column-selection-switch-${selectedColumns[0].field}`)
+      ).toHaveAttribute('aria-checked', selectedColumns[0].isChecked.toString());
 
-    selectedColumns.forEach(({ field, name, isChecked }) => {
-      expect(screen.getByTestId(`column-selection-switch-${field}`)).toHaveAttribute(
-        'aria-checked',
-        isChecked.toString()
-      );
-      expect(screen.getByText(name)).toBeInTheDocument();
+      selectedColumns.slice(1).forEach(({ field, name, isChecked }) => {
+        expect(screen.getByTestId(`column-selection-switch-${field}`)).toHaveAttribute(
```

Comment:
```suggestion
        expect(await screen.findByTestId(`column-selection-switch-${field}`)).toHaveAttribute(
```

and below on line 48 also missing 🙌 

Why not remove the `.slice(1)` and do this for the whole array? we basically already do the same for `selectedColumns[0]` outside the `forEach`.